SOFTWARE-LIST.md

```diff
@@ -2403,6 +2403,8 @@ download | | 12/20/2021 |
 | SmileCDR | | | | | [SmileCDR Blog Post](https://www.smilecdr.com/our-blog/a-statement-on-log4shell-cve-2021-44228) | | | |
 | Snakemake | Snakemake | 6.12.1 | Not Affected |  | [https://snakemake.readthedocs.io/en/stable/](https://snakemake.readthedocs.io/en/stable/) | | | 12/21/2021 |
 | Sn0m | | | | | [Sn0m Link](https://www.snom.com/en/press/log4j-poses-no-threat-snom-phones/) | | | |
+| Snow Software | Snow Commander | 8.1 to 8.10.2 | Fixed | Yes |[Snow Software Commmunity Link](https://community.snowsoftware.com/s/feed/0D5690000B4U6hUCQS) | | | |
+| Snow Software | VM Access Proxy | v3.1 to v3.6 " Fixed | Yes | [Snow Software Commmunity Link](https://community.snowsoftware.com/s/feed/0D5690000B4U6hUCQS) | | | |
```

Comment:
Line should read as the following: 
| Snow Software | VM Access Proxy | v3.1 to v3.6 I Fixed | Yes | [Snow Software Community Link](https://community.snowsoftware.com/s/feed/0D5690000B4U6hUCQS) | | | |